docs/pt/docs/tutorial/cors.md

```diff
@@ -0,0 +1,84 @@
+# CORS (Cross-Origin Resource Sharing)
+
+<a href="https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS" class="external-link" target="_blank">CORS ou "Cross-Origin Resource Sharing"</a> refere-se às situações em que um frontend radando em um navegador possui um código JavaScript que se comunica com um backend, e o backend está em uma "origem" diferente do frontend.
+
+## Origem
+
+Uma origem é a combinação de protocolo (`http`, `https`), domínio (`myapp.com`, `localhost`, `localhost.tiangolo.com`), e porta (`80`, `443`, `8080`).
+
+Então, todos estes são origens diferentes:
+
+* `http://localhost`
+* `https://localhost`
+* `http://localhost:8080`
+
+Mesmo se todos estiverem em `localhost`, eles usam diferentes protocolos e portas, portanto, são "origens" diferentes.
+
+## Passos
+
+Então, digamos que você tenha um frontend rodando no seu navegador em `http://localhost:8080`, e seu JavaScript esteja tentando se comunicar com um backend rodando em http://localhost (como não especificamos uma porta, o navegador assumirá a porta padrão `80`).
+
+Portanto, o navegador irá enviar uma requisição HTTP `OPTIONS` ao backend, e se o backend enviar os cabeçalhos apropriados autorizando a comunicação a partir de uma origem diferente (`http://localhost:8080`) então o navegador deixará o JavaScript no frontend enviar sua requisição para o backend.
+
+Para conseguir isso, o backend deve ter uma lista de "origens permitidas".
+
+Neste caso, ele terá que incluir `http://localhost:8080` paro frontend funcionar corretamente.
+
+## Curingas
+
+É possível declarar uma lista com `"*"` (um "curinga") para dizer que tudo está permitido.
+
+Mas isso só permitirá certos tipos de comunicação, excluindo tudo que envolva credenciais: cookies, cabeçalhos de autorização como aqueles usados ​​com Bearer Tokens, etc.
+
+Então, para que tudo funcione corretamente, é melhor especificar explicitamente as origens permitidas.
+
+## Usar `CORSMiddleware`
+
+Você pode configurá-lo em sua palicação **FastAPI** usando o `CORSMiddleware`.
+
+* Importe `CORSMiddleware`.
+* Crie uma lista de origens permitidas (como strings).
+* Adicione-a como um "middleware" à sua aplicação **FastAPI**.
+
+Você também pode especificar se seu backend permite:
+
+* Credenciais (Cabeçlhos de autorização, Cookies, etc).
+* Métodos HTTP específicos (`POST`, `PUT`) ou todos eles com o curinga `"*"`.
+* Cabeçalhos HTTP específicos ou todos eles com o curinga `"*"`.
+
+```Python hl_lines="2  6-11  13-19"
+{!../../../docs_src/cors/tutorial001.py!}
+```
+
+Os parâmetros padrão usados ​​pela implementação `CORSMiddleware` são restritivos por padrão, então você precisará habilitar explicitamente as origens, métodos ou cabeçalhos específicos para que os navegadores tenham permissão para usá-los em um contexto de domínios diferentes.
+
+Os seguintes argumentos são apoiados:
```

Comment:
I think "suportados" is better then "apoiados" here

Reply:
I totally agree. I'll fix it.